### PR TITLE
Adding result download interaction

### PR DESCRIPTION
**Pull request by gjvera**

### CODE REVIEW

File: src/main/webapp/components/results/result-index-cards.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  const { results, handleOpen, setResult } = props
+  return React.useMemo(
+    () =>
+      results.map(({ metacard }) => (
+        <IndexCardItem
+          key={metacard.attributes.id}
+          title={metacard.attributes.title}
+          subHeader={' '}
+          titleTypographyProps={{
+            variant: 'body1',
```
Comment (gjvera) [inline src/main/webapp/components/results/result-index-cards.js:17]: image
Thoughts on this styling change

File: src/main/webapp/components/workspace/workspace.js [hunk 2]
```diff
@@ -236,7 +210,7 @@ export default () => {
                   })
                 }}
               />
-              <Results
+              <ResultIndexCards
                 results={results}
                 handleOpen={handleOpen}
```
Comment (willwill96) [inline src/main/webapp/components/workspace/workspace.js:215]: :pencil2: I don't think that workspace should care about the logic related to lists.

I think we should move the ResultListInteraction and Popover components as well as the selectedResult state into the ResultIndexCards component(if not it's own).

File: src/main/webapp/components/results/result-index-cards.js (hunk 1 above)
Comment (willwill96) [reply src/main/webapp/components/results/result-index-cards.js:17]: lgtm - on an unrelated note, it kind of looks like the add and download buttons are slightly off center, but that could just be the way the SVGs are

File: src/main/webapp/components/results/result-index-cards.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+import { ResultListInteraction } from '../lists/result-list-interaction'
+
+export default props => {
+  const { results, setLists, lists } = props
+  const [selectedResult, setSelectedResult] = React.useState(null)
+  const [anchorEl, handleOpen, handleClose, isOpen] = useAnchorEl()
+
+  return (
+    <React.Fragment>
+      {' '}
```
Comment (willwill96) [inline src/main/webapp/components/results/result-index-cards.js:17]: should this be deleted?

File: src/main/webapp/components/results/result-index-cards.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                        metacard.attributes['resource-download-url'],
+                        '_blank'
+                      )
+                    }}
+                    size="small"
+                  >
+                    <GetAppIcon />
+                  </IconButton>
+                ) : null}
+                {props.actions ? props.actions : null}
```
Comment (willwill96) [inline src/main/webapp/components/results/result-index-cards.js:52]: :pencil2: could also be written `props.actions || null`

File: src/main/webapp/components/results/result-index-cards.js (hunk 3 above)
Comment (gjvera) [reply src/main/webapp/components/results/result-index-cards.js:17]: I'm not sure why but whenever I format this comes up whenever something is wrapped in a React.Fragment. I'll remove it.

**Pull request merged**